### PR TITLE
`Paywalls`: internal beta setup

### PR DESCRIPTION
**Pull request by NachoSoto**

### CODE REVIEW

File: examples/MagicWeather/app/src/main/java/com/revenuecat/sample/data/Constants.kt [hunk 1]
```diff
@@ -7,15 +7,15 @@ object Constants {
 
      Modify this property to reflect your Google Play app's API key.
      */
-    const val GOOGLE_API_KEY = "googl_api_key"
+    const val GOOGLE_API_KEY = "goog_cRtaFpfwfcQYPXkIEqFUkVszUfN"
```
Comment (tonidero) [inline examples/MagicWeather/app/src/main/java/com/revenuecat/sample/data/Constants.kt:10]: Oops

File: build.gradle [hunk 2]
```diff
@@ -4,6 +4,8 @@ buildscript {
 }
 
 plugins {
+    id 'maven-publish'
```
Comment (tonidero) [inline build.gradle:7]: Hmm I think this shouldn't be needed? The `alias libs.plugins.mavenPublish apply false` adds the plugin without applying, so each plugin needs to apply it later

File: ui/revenuecatui/build.gradle [hunk 3]
```diff
@@ -1,4 +1,6 @@
 plugins {
+    id 'maven-publish'
```
Comment (tonidero) [inline ui/revenuecatui/build.gradle:2]: This shouldn't be needed either I think, just uncommenting the code below should apply the plugin

File: examples/MagicWeather/app/src/main/java/com/revenuecat/sample/data/Constants.kt (hunk 1 above)
Comment (NachoSoto) [reply examples/MagicWeather/app/src/main/java/com/revenuecat/sample/data/Constants.kt:10]: Pushed it on purpose for testers to be able to run it

File: build.gradle (hunk 2 above)
Comment (NachoSoto) [reply build.gradle:7]: Maybe not, but we're not merging this PR anyway.

**Comment by NachoSoto** [comment]
Closing this.

**Pull request closed without merging**